tools/image_processing/yolo/adjust_save_images.py

```diff
@@ -0,0 +1,46 @@
+import argparse
+import os
+
+from skimage import exposure, img_as_ubyte
+from skimage.io import imread, imsave
+
+
+def apply_clahe(image):
```

Comment:
`apply_clahe` is not used here?

Reply:
The function already exists in other existing tool, so I removed the function

Reply:
not referenced. I will remove it. Thanks